### DISCUSSION TITLE
Account Temporarily Suspended - Why?

### DISCUSSION THREAD

**Discussion by stoneymahoney42** [chromiecraft/chromiecraft#2392] [General]
Account: stoneymahoney
Just getting back on to Chromiecraft after an absence, noticed horrific stuttering while running around the world and was trying different things to try and smooth the game out. Tried setting compatibility mode to Windows XP SP3 as I was told that might help - it didn't, but it did seem to get my account banned?

**Comment by 55Honey** [comment]
Hi stoneymahoney,
Please check your video settings regarding `vsync` and `hardware cursor` are they on or off?
What's your GPU manufacturer? Intel? AMD? Nvidia?

Feel free to make a ticket on our discord for support in private.
I've unbanned your account.

**Comment by stoneymahoney42** [comment]
Don't bother. I'll go play a game that doesn't have such egrigiously heavy-handed policies as to ban accounts for their Vsync and hardware cursor settings, ffs.

**Comment by 55Honey** [comment]
An automated account suspension is not strictly related to your settings but to the behaviour caused by them. Your client appeared to the server as if it was intentionally lagging due to mismatching settings of a 11+ year old game on a (most likely) more modern machine.
